test/Npgsql.Tests/ConnectionTests.cs

```diff
@@ -1353,6 +1367,26 @@ public void Connect_PasswordFromEnvironment_Succeeds()
             }
         }
 
+        [Test]
+        [NonParallelizable]
+        public void Connect_OptionsFromEnvironment_Succeeds()
+        {
+            var builder = new NpgsqlConnectionStringBuilder(ConnectionString)
+            {
+                Pooling = false,
+                Options = "default_transaction_isolation=serializable  default_transaction_deferrable=on application_name=My\\ Famous\\\\App"
```

Comment:
Any reason to assign the options string to builder.Options, and later to set it to null? It can be just set directly into the environment variable (or assigned to some local variable)